src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -63,4 +63,11 @@ public boolean isPreviousPageHeadingVisible(String pageHeading) {
         visually.checkWindow(SCREEN_NAME, "Page landed after tapping in the middle");
         return driver.isElementPresent(AppiumBy.xpath(String.format(byPageHeaderXpath, pageHeading)));
     }
+
+    @Override
+    public VodqaScreen validateAppWorkInBackground(int time) {
+        driver.putAppInBackground(time);
+        visually.checkWindow(SCREEN_NAME, "Home screen after putting app in background");
+        return this;
+    }
```

Comment:
method name says validate but validation is not happening here.
validate and make return type as boolean, assert the same in BL

Reply:
resolved